README.md

```diff
@@ -205,6 +205,15 @@ Supporting modules
 | [@gasket/utils]            | 6.24.0  | Reusable utilities for Gasket internals                                              |
 | [create-gasket-app]        | 6.24.1  | starter pack for creating a gasket app                                               |
 
+## Configurations
+
+Available configuration options in the `gasket.config.js`
+
+| Name             | Description                         | Type   | Default | From                |
```

Comment:
I feel like the `From` column crowds the table too much and is unnecessary with the name links. I will defer to other opinions. However, if we do keep the column, it seems like they should link, also.

Reply:
Removed the `From` column in favor of the link fallbacks option(set to true for the configurations generated portion). Links if not specified will point to the package README.